.github/workflows/user-service-ci.yml

```
@@ -0,0 +1,174 @@
+            Clients__sellercenter__IdentityTokenLifetime=7200
+            Authentication__Google__ClientId=${{ secrets.GOOGLE_CLIENT_ID }}
+            Authentication__Google__ClientSecret=${{ secrets.GOOGLE_CLIENT_SECRET }}
+            Duende__LicenseKey=${{ secrets.DUENDE_LICENSE_KEY }}
+            MediatR__LicenseKey=${{ secrets.MEDIATR_LICENSE_KEY }}
+            EmailSettings__SmtpServer=${{ vars.SMTP_SERVER }}
+            EmailSettings__SmtpPort=${{ vars.SMTP_PORT }}
+            EmailSettings__SmtpUser=${{ secrets.SMTP_USER }}
+            EmailSettings__SmtpPassword=${{ secrets.SMTP_PASSWORD }}
+            EmailSettings__FromEmail=${{ vars.FROM_EMAIL }}
+            EmailSettings__FromName=${{ vars.FROM_NAME }}
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)